hmpps-clamav/Dockerfile

```diff
@@ -3,9 +3,12 @@ FROM clamav/clamav:stable_base
 # Install kubectl - needed for our daily reload cronjob
 ENV KUBECTL_VERSION=v1.20.15
 RUN apk update && \
-    apk upgrade && \
-  apk add --no-cache curl && \
-  curl -LO "https://dl.k8s.io/release/$KUBECTL_VERSION/bin/linux/amd64/kubectl" && \
+    apk upgrade
+
+# Remediation: CVE-2023-38545
+RUN apk add --no-cache 'curl>8.4.0'
```

Comment:
This should be run as part of the `RUN` above to avoid creating unnecessary layers in the container image.